From e58bc3859081f832a6aab88c0abac2c5297c2b29 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 30 Oct 2016 19:09:09 +0200
Subject: [PATCH 0626/1051] ap806: dts: add revision info node

A8K requires different handling for various chip revisions.
This patch adds a node allowing to get the SoC revision in runtime.

Change-Id: I8899cea3c37a727bd93ab2d392bdcb445789ee53
Signed-off-by: Igal Liberman <igall@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-ap806.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
index 24f99a8acbaa..985eb59dfd22 100644
--- a/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-ap806.dtsi
@@ -272,6 +272,11 @@
 				reg = <0x6f808c 0x4>,
 				      <0x6f8084 0x8>;
 			};
+
+			revision-info@610fcc {
+				compatible = "marvell,ap806-rev-info";
+				reg = <0x610fcc 0x4>;
+			};
 		};
 	};
 };
-- 
2.17.1

